src/components/CustomMarker/getImage.js

```diff
@@ -0,0 +1,45 @@
+import art from 'assets/images/art.png';
+import dating from 'assets/images/dating.png';
+import food from 'assets/images/food.png';
+import football from 'assets/images/football.png';
+import movies from 'assets/images/movies.png';
+import music from 'assets/images/music.png';
+import politics from 'assets/images/politics.png';
+import series from 'assets/images/series.png';
+import world from 'assets/images/world.png';
+
+export function getImage(topicId) {
```

Comment:
What do you think about exporting an object with the images, and base by key you get the value

Reply:
That would be good, but read Manuelas comment.